easybuild/tools/module_generator.py

```diff
@@ -998,6 +1009,15 @@ def __init__(self, *args, **kwargs):
             if self.modules_tool.version and LooseVersion(self.modules_tool.version) >= LooseVersion('7.7.38'):
                 self.DOT_MODULERC = '.modulerc.lua'
 
+    def check_version(self, minimal_version_maj, minimal_version_min):
+        """
+        Check the minimal version of the moduletool in the module file
+        """
+        return self.VERSION_CHECK % {
```

Comment:
if `VERSION_CHECK` is only used in one place, I'm not sure it's worth adding a class constant for, just embed the `VERSION_CHECK` in this method (which could be a static method btw)